### ISSUE TITLE
Duplicate entries when running multiple times.

### ISSUE THREAD

**Issue by valeriomazzeo** [kareman/linuxmain-generator#1]
Running linuxmain-generator multiple times cause the `allTests` extension to be added multiple times.

Expected behaviour would be to remove the previous extension and rewrite the new one.

**Comment by kareman** [comment]
I agree. This basically requires parsing the Swift code, making sure we don't remove the wrong code. I'm busy with other projects at the moment, so any pull requests are most welcome.
